gym/wrappers/monitoring/tests/test_video_recorder.py

```diff
@@ -5,7 +5,7 @@
 import numpy as np
 
 import gym
-from gym.wrappers.monitoring.video_recorder import VideoRecorder
+from gym.wrappers.monitoring.video_recorder import VideoRecorder, video_recorder_closer
```

Comment:
I don't think this new import is used

Reply:
Yes, and this could be removed.

This was imported for the draft version of `test_autoclose` (https://github.com/openai/gym/commit/e8e81d8c441c88df6a1225fe9fcd31b663183fb7), and the test case had been updated as suggested in https://github.com/openai/gym/pull/2318#issuecomment-899923137.

Reply:
Could you please great a small PR adressing this and the other note?

Reply:
I have opened a PR #2404 to test for the number of recorders that registered in the closer.